mrscake/src/components/SignUp.jsx

```diff
@@ -108,27 +108,27 @@ class SignUp extends Component {
                     <div>
 
                     <FormGroup className="mb-3">
-                    <Input required='true' type="text" name="firstName" onChange={this.handleChange} value={this.state.firstname === '' ? '' : this.state.firstName} placeholder="First Name" />
```

Comment:
I'm sure we need at least some of these field (email, password, user role)  required